lib/tapioca/compilers/dsl/active_record_columns.rb

```diff
@@ -347,6 +347,21 @@ def do_not_generate_strong_types?(constant)
           Object.const_defined?(:StrongTypeGeneration) &&
               !(constant.singleton_class < Object.const_get(:StrongTypeGeneration))
         end
+
+        def handle_unknown_type(column_type)
+          if column_type.ancestors.include?(ActiveModel::Type::Value)
+            signature = T::Private::Methods.signature_for_method(column_type.instance_method(:deserialize))
+            return signature.return_type.to_s if signature
```

Comment:
We need to be a little bit more careful with return types since they can be `T::Private::Types::Void` or `T::Private::Types::NotTyped` both of which are internal Sorbet types.

It might also be nice to extract two methods from here so that this method can just be:
```ruby
def handle_unknown_type(column_type)
  return "T.unyped" unless column_type < ActiveModel::Type::Value

  lookup_return_type_of_method(column_type, :deserialize) ||
    lookup_return_type_of_method(column_type, :cast) ||
    lookup_arg_type_of_method(column_type, :serialize) ||
    "T.untyped"
end
```